public/js/login.js

```diff
@@ -0,0 +1,32 @@
+const loginFormHandler = async (event) => {
+  event.preventDefault();
+
+  // trim = leerzeichen vorne und hinten abschneiden
+  const username = document.querySelector("#username").value.trim();
+  const password = document.querySelector("#password").value.trim();
```

Comment:
great addition of the trim method! Makes your string tidy and easier to manipulate :-)